### PR TITLE
Revert "Revert "Sass 3250""

### PR DESCRIPTION
**Pull request by abiodxn**
Reverts hmrc/income-tax-submission#49

Necessary work has taken place to breaks that were previously caused by this PR.

### CODE REVIEW

**Comment by erasmos-hmrc** [comment]
@abiodxn , could you amend the commit message to contain the parent key (SASS-3248), and describe it?

**Review by salman-a-dev — DISMISSED** [review]

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -19,7 +19,7 @@ import sbt._
 
 object AppDependencies {
 
-  private val bootstrapPlay28Version = "6.4.0"
+  private val bootstrapPlay28Version = "7.0.0"
```
Comment (erasmos-hmrc) [inline project/AppDependencies.scala:22]: Why was this updated?

Comment (abiodxn) [reply project/AppDependencies.scala:22]: It was updated as it was out of sync with the app dependencies before the revert

Comment (erasmos-hmrc) [reply project/AppDependencies.scala:22]: OK

**Review by salman-a-dev — APPROVED** [review]

**Pull request merged**